modules/eco-node-maintenance-operator-installation-web-console.adoc

```diff
@@ -29,4 +29,5 @@ To confirm that the installation is successful:
 If the Operator is not installed successfully:
 
 . Navigate to the *Operators* -> *Installed Operators* page and inspect the `Status` column for any errors or failures.
-. Navigate to the *Workloads* -> *Pods* page and check the logs in any pods in the `openshift-operators` project that are reporting issues.
+. Navigate to the *Operators* -> *Installed Operators* -> *Node-maintenance-operator Details* page and inspect errors in the `Conditions` sections. There may be errors before pods were created.
```

Comment:
Thanks @anna-savina , updated now

[Installing the Node Maintenance Operator by using the web console](http://file.emea.redhat.com/pogrady/TELCODOCS-28/nodes/nodes/eco-node-maintenance-operator.html#installing-node-maintenance-operator-using-web-console_node-maintenance-operator)

Reply:
1. @ogradyp  I don't see your change in https://github.com/openshift/openshift-docs/pull/46914/files#diff-d40517f6dbef6eeac9fc5d33b3c43b566f531a4c384a7fb4b1fa8533e1130ca3R32 for [About the Node Maintenance Operator](http://file.emea.redhat.com/pogrady/TELCODOCS-28/nodes/nodes/eco-node-maintenance-operator.html#eco-about-node-maintenance-operator_node-maintenance-operator).
2. Also there ate two additional plus signs (+) and one dot sign (.) that I believe we can dismiss, see https://github.com/openshift/openshift-docs/pull/46914/files#diff-617d99b26aae91621ad418181458913442c3d74c707503f4a1bb88ff05d82886R43 for [Setting a node to maintenance mode by using the CLI](http://file.emea.redhat.com/pogrady/TELCODOCS-28/nodes/nodes/eco-node-maintenance-operator.html#eco-setting-node-maintenance-cr-cli_node-maintenance-operator).
3. How about writing it as one line (pause) and then the command rather than one line (pause), command(pause), and a new line (which is part of the original line..)?
Also, it might be better to use a one method/command for verification that the node is (or not) under maintenance for [Setting a node to maintenance mode by using the CLI](http://file.emea.redhat.com/pogrady/TELCODOCS-28/nodes/nodes/eco-node-maintenance-operator.html#eco-setting-node-maintenance-cr-cli_node-maintenance-operator) & [Resuming a node from maintenance mode by using the CLI](http://file.emea.redhat.com/pogrady/TELCODOCS-28/nodes/nodes/eco-node-maintenance-operator.html#eco-resuming-node-from-maintenance-mode-with-cr_node-maintenance-operator).
`oc describe node <node-name>` sounds very good to me.
@anna-savina WDYT?
If so then I would pretty much copy paste the verification method from [Setting a node to maintenance mode by using the CLI](http://file.emea.redhat.com/pogrady/TELCODOCS-28/nodes/nodes/eco-node-maintenance-operator.html#eco-setting-node-maintenance-cr-cli_node-maintenance-operator), without the dot and plus signs, and only add a new `Example output`.